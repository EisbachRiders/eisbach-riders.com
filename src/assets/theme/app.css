html {
  height: 100%;
}

body {
  margin: 0;
  padding: 0;
  font-family: 'Raleway', sans-serif;
  background-color: #ffffff;
  overflow-x: hidden;
}

a {
  text-decoration: none;
}
