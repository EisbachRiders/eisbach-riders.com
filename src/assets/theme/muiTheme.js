import { createMuiTheme } from '@material-ui/core/styles'
import './App.css'
require('typeface-open-sans')
require('typeface-contrail-one')

const status = {
  white: '#ffffff',
  offWhite: '#F7F5F5',
  black: '#0b0c10',
  blue: '#1f2833',
  grey: '#c5c6c7',
  greyDk: '#404040',
  teal: '#66fcf1',
  tealDk: '#45a29e',
  green: '#66fca6',
  yellow: '#fcf166',
  red: '#e25b65',
}

const theme = createMuiTheme({
  palette: {
    type: 'light',
    primary: {
      light: status.teal,
      main: status.teal,
      dark: status.tealDk,
    },
    secondary: {
      light: status.blue,
      main: status.blue,
      dark: status.blue,
    },
    error: { main: status.red },
  },
  status,
  typography: {
    fontFamily: "'Open Sans', sans-serif",
    fontSize: 14,
  },
  overrides: {
    MuiTypography: {
      colorPrimary: {
        color: status.greyDk,
      },
      colorSecondary: {
        color: status.grey,
      },
      colorTextSecondary: {
        color: status.white,
      },
    },
  },
})

export default theme
