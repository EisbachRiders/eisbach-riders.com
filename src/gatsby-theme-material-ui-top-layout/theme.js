import { createMuiTheme } from "@material-ui/core/styles"
import "../app.css"
import "../../static/fonts/fonts.css"

// fin: "#00d7a2",
// finDk: "#00be8f",
// finLt: "#e2f9ef",
// greyLt2: "#eeeff0",
// greyLt: "#d5d9db",
// grey: "#818C93",
// greyBlue: "#f3f8fc",
// greenLt1: "#6dff97",
// greenLt2: "#00cf3c",
// cream: "#f7f5ee",

const color = {
  // for shop colors
  blue: "#004AB5",
  green: "#00942b",
  yellow: "#fce566",
  orange: "#ff8000",
  pink: "#ff0080",
  red: "#FF0012",
  redLt: "#ff4e5b",
  // theme colors
  white: "#ffffff",
  black: "#000000",
  almostBlack: "#363738", // typography
  textLight: "#838588",
  gray: "#ebedee",
  primaryBlue: "#0062f0",
  primaryBlueLt: "#3f8eff",
  gold: "#E6C55E",
  goldLt: "#f0dda0",
}

const theme = createMuiTheme({
  palette: {
    type: "light",
    primary: {
      light: color.goldLt,
      main: color.gold,
      dark: color.goldLt,
    },
    secondary: {
      light: color.primaryBlueLt,
      main: color.primaryBlue,
      dark: color.primaryBlueLt,
    },
    error: { main: color.red },
  },
  color,
  typography: {
    fontFamily: "'Raleway', sans-serif",
    h1: {
      letterSpacing: 2,
    },
    h2: {
      letterSpacing: 2,
    },
    h3: {
      letterSpacing: 2,
    },
    h4: {
      letterSpacing: 2,
    },
    h5: {
      letterSpacing: 2,
    },
    body1: {
      letterSpacing: 1.5,
      fontSize: 14,
    },
    body2: {
      letterSpacing: 1.5,
      fontSize: 12,
    },
    button: {
      fontFamily: "'Raleway', sans-serif",
      letterSpacing: 3,
    },
  },
  overrides: {
    MuiButton: {
      root: {
        padding: "9px 27px",
        borderRadius: 0,
      },
      contained: {
        boxShadow: "none",
      },
    },
  },
})

export default theme
