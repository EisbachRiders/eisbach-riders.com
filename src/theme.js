import { createTheme } from "@material-ui/core/styles"
import "./css/app.css"

// fin: "#00d7a2",
// finDk: "#00be8f",
// finLt: "#e2f9ef",
// grayLt2: "#eeeff0",
// grayLt: "#d5d9db",
// gray: "#818C93",
// grayBlue: "#f3f8fc",
// greenLt1: "#6dff97",
// greenLt2: "#00cf3c",
// cream: "#f7f5ee",

const color = {
  // for shop colors
  blue: "#004AB5",
  green: "#00942b",
  yellow: "#fce566",
  orange: "#ff8000",
  pink: "#ff0080",
  red: "#FF0012",
  redLt: "#ff4e5b",
  seaGlass: "#5F5850",
  // theme colors
  white: "#ffffff",
  black: "#0B0C10",
  blackLt: "#1F2833",
  gray: "#C5C6C7",
  // teal: "#66FCF1",
  teal: "#00d7a2",
  tealDk: "#45A29E",
  test: "#F7F680",
  cream: "#F1F0EB",
  grayLt: "#f7f7f9",
  blueGray: "#cbd3cf",
  steel: "#505D72",
}

const theme = createTheme({
  palette: {
    type: "light",
    primary: {
      light: color.tealDk,
      main: color.teal,
      dark: color.tealDk,
    },
    secondary: {
      light: "#9d9fa0",
      main: color.gray,
      dark: "#9d9fa0",
    },
    error: { main: color.red },
  },
  color,
  typography: {
    fontFamily: ["Raleway", "sans-serif"].join(","),
  },
  components: {
    MuiButton: {
      styleOverrides: {
        root: {
          padding: "9px 27px",
          borderRadius: 0,
        },
        contained: {
          boxShadow: "none",
        },
      },
    },
  },
})

export default theme
