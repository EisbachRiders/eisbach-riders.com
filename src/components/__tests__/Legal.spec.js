import React from 'react'
import { render } from 'react-testing-library'
import Legal from '../Legal'
import { ThemeProvider } from '@material-ui/styles'
import theme from '../../theme'

describe('Legal', () => {
  it('renders correctly', () => {
    const mockProps = {
      t: () => '',
    }
    const component = render(
      <ThemeProvider theme={theme}>
        <Legal {...mockProps} />
      </ThemeProvider>
    )
    expect(component).toMatchSnapshot()
  })
})
