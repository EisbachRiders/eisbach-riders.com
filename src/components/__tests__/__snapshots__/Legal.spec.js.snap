// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Legal renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="Hook-container-6t7not"
        id="legal"
      >
        <h5
          class="MuiTypography-root MuiTypography-h5 Hook-title-j4ku9t"
        >
          imprint.imprint
        </h5>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          Eisbach Riders 
          <br />
          Robin Eisenhardt & Michael Schmidt GbR 
          <br />
          Lothstr. 72 
          <br />
          80797 München 
          <br />
          Deutschland 
          <br />
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          Tel.: 0176/69019542 
          <br />
          E-Mail: business@eisbach-riders.com
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          Vertretungsberechtigte Gesellschafter: Robin Eisenhardt, Michael Schmidt
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          Umsatzsteuer-Identifikationsnummer gemäß §27 a Umsatzsteuergesetz: DE 320071401 Plattform der EU-Kommission zur 
          <br />
          Online-Streitbeilegung:
           
          <a
            href="https://ec.europa.eu/consumers/odr"
          >
            https://ec.europa.eu/consumers/odr
          </a>
          Wir sind zur Teilnahme an einem Streitbeilegungsverfahren vor einer Verbraucherschlichtungsstelle weder verpflichtet noch bereit.
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          Verantwortliche/r i.S.d. § 55 Abs. 2 RStV: Robin Eisenhardt, Michael Schmidt, Lothstr. 72, 80797, München
        </p>
        <h6
          class="MuiTypography-root MuiTypography-h6 Hook-title-j4ku9t"
        >
          imprint.terms
        </h6>
        <h6
          class="MuiTypography-root MuiTypography-subtitle1"
        >
          imprint.title1
        </h6>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          imprint.terms1_1
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          imprint.terms1_2
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          imprint.terms1_3
        </p>
        <h6
          class="MuiTypography-root MuiTypography-subtitle1"
        >
          imprint.title2
        </h6>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          imprint.terms2_1
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          imprint.terms2_2
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          imprint.terms2_3
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          imprint.terms2_4
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          imprint.terms2_5
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          imprint.terms2_6
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          imprint.terms2_7
        </p>
        <h6
          class="MuiTypography-root MuiTypography-subtitle1"
        >
          imprint.title3
        </h6>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          imprint.terms3_1
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          imprint.terms3_2
        </p>
        <h6
          class="MuiTypography-root MuiTypography-subtitle1"
        >
          imprint.title4
        </h6>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          imprint.terms4_1
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          imprint.terms4_2
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          imprint.terms4_3
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          imprint.terms4_4
        </p>
        <h6
          class="MuiTypography-root MuiTypography-subtitle1"
        >
          imprint.title5
        </h6>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          imprint.terms5_1
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          imprint.terms5_2
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          imprint.terms5_3
        </p>
        <h6
          class="MuiTypography-root MuiTypography-subtitle1"
        >
          imprint.title6
        </h6>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          imprint.terms6_1
        </p>
        <h6
          class="MuiTypography-root MuiTypography-subtitle1"
        >
          imprint.title7
        </h6>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          imprint.terms7_1
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          imprint.terms7_2
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          imprint.terms7_3
        </p>
        <h6
          class="MuiTypography-root MuiTypography-subtitle1"
        >
          imprint.title8
        </h6>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          imprint.terms8_1
        </p>
        <h6
          class="MuiTypography-root MuiTypography-subtitle1"
        >
          imprint.title9
        </h6>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          imprint.terms9_1
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          imprint.terms9_2
        </p>
        <h6
          class="MuiTypography-root MuiTypography-h6 Hook-title-j4ku9t"
        >
          imprint.title10
        </h6>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          imprint.terms10_1
        </p>
        <h6
          class="MuiTypography-root MuiTypography-subtitle1"
        >
          imprint.title11
        </h6>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          imprint.terms11_1
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          imprint.terms11_2
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          imprint.terms11_3
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          imprint.terms11_4
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          imprint.terms11_5
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          imprint.terms11_6
        </p>
        <h6
          class="MuiTypography-root MuiTypography-subtitle1"
        >
          imprint.title12
        </h6>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          imprint.terms12_1
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          Eisbach Riders 
          <br />
          Robin Eisenhardt & Michael Schmidt GbR 
          <br />
          Lothstr. 72 
          <br />
          80797 München 
          <br />
          Deutschland 
          <br />
          E-Mail: business@eisbach-riders.com
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
        >
          imprint.terms12_2
        </p>
        <h5
          class="MuiTypography-root MuiTypography-h5 Hook-title-j4ku9t"
        >
          imprint.credits
        </h5>
        <a
          href="https://unsplash.com/@thefoxicon?utm_medium=referral&utm_campaign=photographer-credit&utm_content=creditBadge"
          rel="noopener noreferrer"
          style="background-color: black; color: white; text-decoration: none; padding: 4px 6px; font-family: -apple-system, BlinkMacSystemFont, \\"San Francisco\\", \\"Helvetica Neue\\", Helvetica, Ubuntu, Roboto, Noto, \\"Segoe UI\\", Arial, sans-serif; font-size: 12px; font-weight: bold; line-height: 1.2; display: inline-block; border-radius: 3px;"
          target="_blank"
          title="Download free do whatever you want high-resolution photos from Mike Fox"
        >
          <span
            style="display: inline-block; padding: 2px 3px;"
          >
            <svg
              style="height: 12px; width: auto; position: relative; vertical-align: middle; top: -2px; fill: white;"
              viewBox="0 0 32 32"
              xmlns="http://www.w3.org/2000/svg"
            >
              <title>
                unsplash-logo
              </title>
              <path
                d="M10 9V0h12v9H10zm12 5h10v18H0V14h10v9h12v-9z"
              />
            </svg>
          </span>
          <span
            style="display: inline-block; padding: 2px 3px;"
          >
            Mike Fox
          </span>
        </a>
        <a
          href="https://unsplash.com/@jeremybishop?utm_medium=referral&utm_campaign=photographer-credit&utm_content=creditBadge"
          rel="noopener noreferrer"
          style="background-color: black; color: white; text-decoration: none; padding: 4px 6px; font-family: -apple-system, BlinkMacSystemFont, \\"San Francisco\\", \\"Helvetica Neue\\", Helvetica, Ubuntu, Roboto, Noto, \\"Segoe UI\\", Arial, sans-serif; font-size: 12px; font-weight: bold; line-height: 1.2; display: inline-block; border-radius: 3px;"
          target="_blank"
          title="Download free do whatever you want high-resolution photos from Jeremy Bishop"
        >
          <span
            style="display: inline-block; padding: 2px 3px;"
          >
            <svg
              style="height: 12px; width: auto; position: relative; vertical-align: middle; top: -2px; fill: white;"
              viewBox="0 0 32 32"
              xmlns="http://www.w3.org/2000/svg"
            >
              <title>
                unsplash-logo
              </title>
              <path
                d="M10 9V0h12v9H10zm12 5h10v18H0V14h10v9h12v-9z"
              />
            </svg>
          </span>
          <span
            style="display: inline-block; padding: 2px 3px;"
          >
            Jeremy Bishop
          </span>
        </a>
        <a
          href="https://unsplash.com/@timmarshall?utm_medium=referral&utm_campaign=photographer-credit&utm_content=creditBadge"
          rel="noopener noreferrer"
          style="background-color: black; color: white; text-decoration: none; padding: 4px 6px; font-family: -apple-system, BlinkMacSystemFont, \\"San Francisco\\", \\"Helvetica Neue\\", Helvetica, Ubuntu, Roboto, Noto, \\"Segoe UI\\", Arial, sans-serif; font-size: 12px; font-weight: bold; line-height: 1.2; display: inline-block; border-radius: 3px;"
          target="_blank"
          title="Download free do whatever you want high-resolution photos from Tim Marshall"
        >
          <span
            style="display: inline-block; padding: 2px 3px;"
          >
            <svg
              style="height: 12px; width: auto; position: relative; vertical-align: middle; top: -2px; fill: white;"
              viewBox="0 0 32 32"
              xmlns="http://www.w3.org/2000/svg"
            >
              <title>
                unsplash-logo
              </title>
              <path
                d="M10 9V0h12v9H10zm12 5h10v18H0V14h10v9h12v-9z"
              />
            </svg>
          </span>
          <span
            style="display: inline-block; padding: 2px 3px;"
          >
            Tim Marshall
          </span>
        </a>
        <div>
          Icons made by
           
          <a
            href="https://www.freepik.com/"
            title="Freepik"
          >
            Freepik
          </a>
           
          from
           
          <a
            href="https://www.flaticon.com/"
            title="Flaticon"
          >
            www.flaticon.com
          </a>
           
          is licensed by
           
          <a
            href="http://creativecommons.org/licenses/by/3.0/"
            target="_blank"
            title="Creative Commons BY 3.0"
          >
            CC 3.0 BY
          </a>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="Hook-container-6t7not"
      id="legal"
    >
      <h5
        class="MuiTypography-root MuiTypography-h5 Hook-title-j4ku9t"
      >
        imprint.imprint
      </h5>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        Eisbach Riders 
        <br />
        Robin Eisenhardt & Michael Schmidt GbR 
        <br />
        Lothstr. 72 
        <br />
        80797 München 
        <br />
        Deutschland 
        <br />
      </p>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        Tel.: 0176/69019542 
        <br />
        E-Mail: business@eisbach-riders.com
      </p>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        Vertretungsberechtigte Gesellschafter: Robin Eisenhardt, Michael Schmidt
      </p>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        Umsatzsteuer-Identifikationsnummer gemäß §27 a Umsatzsteuergesetz: DE 320071401 Plattform der EU-Kommission zur 
        <br />
        Online-Streitbeilegung:
         
        <a
          href="https://ec.europa.eu/consumers/odr"
        >
          https://ec.europa.eu/consumers/odr
        </a>
        Wir sind zur Teilnahme an einem Streitbeilegungsverfahren vor einer Verbraucherschlichtungsstelle weder verpflichtet noch bereit.
      </p>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        Verantwortliche/r i.S.d. § 55 Abs. 2 RStV: Robin Eisenhardt, Michael Schmidt, Lothstr. 72, 80797, München
      </p>
      <h6
        class="MuiTypography-root MuiTypography-h6 Hook-title-j4ku9t"
      >
        imprint.terms
      </h6>
      <h6
        class="MuiTypography-root MuiTypography-subtitle1"
      >
        imprint.title1
      </h6>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        imprint.terms1_1
      </p>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        imprint.terms1_2
      </p>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        imprint.terms1_3
      </p>
      <h6
        class="MuiTypography-root MuiTypography-subtitle1"
      >
        imprint.title2
      </h6>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        imprint.terms2_1
      </p>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        imprint.terms2_2
      </p>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        imprint.terms2_3
      </p>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        imprint.terms2_4
      </p>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        imprint.terms2_5
      </p>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        imprint.terms2_6
      </p>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        imprint.terms2_7
      </p>
      <h6
        class="MuiTypography-root MuiTypography-subtitle1"
      >
        imprint.title3
      </h6>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        imprint.terms3_1
      </p>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        imprint.terms3_2
      </p>
      <h6
        class="MuiTypography-root MuiTypography-subtitle1"
      >
        imprint.title4
      </h6>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        imprint.terms4_1
      </p>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        imprint.terms4_2
      </p>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        imprint.terms4_3
      </p>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        imprint.terms4_4
      </p>
      <h6
        class="MuiTypography-root MuiTypography-subtitle1"
      >
        imprint.title5
      </h6>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        imprint.terms5_1
      </p>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        imprint.terms5_2
      </p>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        imprint.terms5_3
      </p>
      <h6
        class="MuiTypography-root MuiTypography-subtitle1"
      >
        imprint.title6
      </h6>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        imprint.terms6_1
      </p>
      <h6
        class="MuiTypography-root MuiTypography-subtitle1"
      >
        imprint.title7
      </h6>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        imprint.terms7_1
      </p>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        imprint.terms7_2
      </p>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        imprint.terms7_3
      </p>
      <h6
        class="MuiTypography-root MuiTypography-subtitle1"
      >
        imprint.title8
      </h6>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        imprint.terms8_1
      </p>
      <h6
        class="MuiTypography-root MuiTypography-subtitle1"
      >
        imprint.title9
      </h6>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        imprint.terms9_1
      </p>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        imprint.terms9_2
      </p>
      <h6
        class="MuiTypography-root MuiTypography-h6 Hook-title-j4ku9t"
      >
        imprint.title10
      </h6>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        imprint.terms10_1
      </p>
      <h6
        class="MuiTypography-root MuiTypography-subtitle1"
      >
        imprint.title11
      </h6>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        imprint.terms11_1
      </p>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        imprint.terms11_2
      </p>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        imprint.terms11_3
      </p>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        imprint.terms11_4
      </p>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        imprint.terms11_5
      </p>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        imprint.terms11_6
      </p>
      <h6
        class="MuiTypography-root MuiTypography-subtitle1"
      >
        imprint.title12
      </h6>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        imprint.terms12_1
      </p>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        Eisbach Riders 
        <br />
        Robin Eisenhardt & Michael Schmidt GbR 
        <br />
        Lothstr. 72 
        <br />
        80797 München 
        <br />
        Deutschland 
        <br />
        E-Mail: business@eisbach-riders.com
      </p>
      <p
        class="MuiTypography-root MuiTypography-body2 Hook-text-qkvoh6"
      >
        imprint.terms12_2
      </p>
      <h5
        class="MuiTypography-root MuiTypography-h5 Hook-title-j4ku9t"
      >
        imprint.credits
      </h5>
      <a
        href="https://unsplash.com/@thefoxicon?utm_medium=referral&utm_campaign=photographer-credit&utm_content=creditBadge"
        rel="noopener noreferrer"
        style="background-color: black; color: white; text-decoration: none; padding: 4px 6px; font-family: -apple-system, BlinkMacSystemFont, \\"San Francisco\\", \\"Helvetica Neue\\", Helvetica, Ubuntu, Roboto, Noto, \\"Segoe UI\\", Arial, sans-serif; font-size: 12px; font-weight: bold; line-height: 1.2; display: inline-block; border-radius: 3px;"
        target="_blank"
        title="Download free do whatever you want high-resolution photos from Mike Fox"
      >
        <span
          style="display: inline-block; padding: 2px 3px;"
        >
          <svg
            style="height: 12px; width: auto; position: relative; vertical-align: middle; top: -2px; fill: white;"
            viewBox="0 0 32 32"
            xmlns="http://www.w3.org/2000/svg"
          >
            <title>
              unsplash-logo
            </title>
            <path
              d="M10 9V0h12v9H10zm12 5h10v18H0V14h10v9h12v-9z"
            />
          </svg>
        </span>
        <span
          style="display: inline-block; padding: 2px 3px;"
        >
          Mike Fox
        </span>
      </a>
      <a
        href="https://unsplash.com/@jeremybishop?utm_medium=referral&utm_campaign=photographer-credit&utm_content=creditBadge"
        rel="noopener noreferrer"
        style="background-color: black; color: white; text-decoration: none; padding: 4px 6px; font-family: -apple-system, BlinkMacSystemFont, \\"San Francisco\\", \\"Helvetica Neue\\", Helvetica, Ubuntu, Roboto, Noto, \\"Segoe UI\\", Arial, sans-serif; font-size: 12px; font-weight: bold; line-height: 1.2; display: inline-block; border-radius: 3px;"
        target="_blank"
        title="Download free do whatever you want high-resolution photos from Jeremy Bishop"
      >
        <span
          style="display: inline-block; padding: 2px 3px;"
        >
          <svg
            style="height: 12px; width: auto; position: relative; vertical-align: middle; top: -2px; fill: white;"
            viewBox="0 0 32 32"
            xmlns="http://www.w3.org/2000/svg"
          >
            <title>
              unsplash-logo
            </title>
            <path
              d="M10 9V0h12v9H10zm12 5h10v18H0V14h10v9h12v-9z"
            />
          </svg>
        </span>
        <span
          style="display: inline-block; padding: 2px 3px;"
        >
          Jeremy Bishop
        </span>
      </a>
      <a
        href="https://unsplash.com/@timmarshall?utm_medium=referral&utm_campaign=photographer-credit&utm_content=creditBadge"
        rel="noopener noreferrer"
        style="background-color: black; color: white; text-decoration: none; padding: 4px 6px; font-family: -apple-system, BlinkMacSystemFont, \\"San Francisco\\", \\"Helvetica Neue\\", Helvetica, Ubuntu, Roboto, Noto, \\"Segoe UI\\", Arial, sans-serif; font-size: 12px; font-weight: bold; line-height: 1.2; display: inline-block; border-radius: 3px;"
        target="_blank"
        title="Download free do whatever you want high-resolution photos from Tim Marshall"
      >
        <span
          style="display: inline-block; padding: 2px 3px;"
        >
          <svg
            style="height: 12px; width: auto; position: relative; vertical-align: middle; top: -2px; fill: white;"
            viewBox="0 0 32 32"
            xmlns="http://www.w3.org/2000/svg"
          >
            <title>
              unsplash-logo
            </title>
            <path
              d="M10 9V0h12v9H10zm12 5h10v18H0V14h10v9h12v-9z"
            />
          </svg>
        </span>
        <span
          style="display: inline-block; padding: 2px 3px;"
        >
          Tim Marshall
        </span>
      </a>
      <div>
        Icons made by
         
        <a
          href="https://www.freepik.com/"
          title="Freepik"
        >
          Freepik
        </a>
         
        from
         
        <a
          href="https://www.flaticon.com/"
          title="Flaticon"
        >
          www.flaticon.com
        </a>
         
        is licensed by
         
        <a
          href="http://creativecommons.org/licenses/by/3.0/"
          target="_blank"
          title="Creative Commons BY 3.0"
        >
          CC 3.0 BY
        </a>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllBySelectText": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getAllByValue": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getBySelectText": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "getByValue": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllBySelectText": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryAllByValue": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryBySelectText": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "queryByValue": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
